iidm/iidm-api/src/test/java/com/powsybl/iidm/network/util/BranchTest.java

```diff
@@ -0,0 +1,296 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.util;
+
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.*;
+import org.junit.Test;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+
+import static junit.framework.TestCase.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ */
+public class BranchTest {
+
+    @Test
+    public void testPQI() {
+        Terminal t1 = mock(Terminal.class);
+        Terminal t2 = mock(Terminal.class);
+        Branch branch = new Branch() {
```

Comment:
Might be shorter to use a Mock + `thenCallRealMethod()`. Not sure it really has some added value to be honest ...